foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/credentials/internal/CredentialMessages.properties

```diff
@@ -18,3 +18,14 @@ SelectDomain=Please select a domain.
 UsernameCannotBeBlank=Username cannot be blank.
 PasswordCannotBeBlank=Password cannot be blank.
 UsernameAlreadyExists=Username {0} already exists for domain {1}.
+
+# For use in the credential prompter
+OK=OK
+Cancel=Cancel
+CredentialPrompterTitle=Password Required
+DescriptionSectionTitle=Description
+DescriptionSectionContent=<form><p>A password is required for the username {0} and the credential domain {1}</p></form>
+PasswordSectionTitle=Please enter a password
+PasswordLabel2=&Password
+ShowPasswordLabel=&Show Password
```

Comment:
This is for a button, so yes, sentence-style capitalization is appropriate. good call.